app/serializers/event_serializer.rb

```diff
@@ -5,7 +5,7 @@ class EventSerializer < ActiveModel::Serializer
 
   def date
     t = object.start_time
-    t.blank? ? '' : %{ #{l t, format: :short}#{t.formatted_offset(false)} }
+    t.blank? ? '' : %{ #{I18n.l t, format: :short}#{t.formatted_offset(false)} }
```

Comment:
@nishanthvijayan I would recommend to look at the very last comment in #904 about that case